### DISCUSSION TITLE
Roadmap

### DISCUSSION THREAD

**Discussion by prypyet** [zen-browser/desktop#3707] [Ideas]
Hi! This is a new project with a lot going on and in my opinion giving the users a roadmap of what features are currently in development will help a lot with coherence! Like the single toolbar feature came out of nowhere! and i still dont know when tab folders are going to come out despite them being teased on the website even a week ago. Having a roadmap about where this project is going, on the zen browser website is extremely crucial and necessary imo.

**Comment by adminy** [comment]
Team seems to be using [github projects](https://github.com/zen-browser/desktop/projects), so perhaps milestones will help.

I'd like to point out that a lot of browser features are not there ... before any features are added, perhaps focus on stability and just completeness.

![image](https://github.com/user-attachments/assets/062929f0-0859-4d08-b822-1f4efc20a906)

**Reply by vraravam** [reply]
that's what's already been announced as the focus by the dev(s)
